content/docs/start/index.md

```diff
@@ -54,31 +54,44 @@ $ git commit -m "Initialize DVC"
 
 Now you're ready to DVC!
 
-The value of DVC's several feature sets is best understood from different
-angles. Pick one of the two trails below to learn about DVC from that
-perspective:
+## Following This Guide
 
-## Data
+To help you understand and use DVC better, consider those two high level
+scenarios:
 
-- **[Data and model versioning]** is the base layer of DVC for large files,
-  datasets, and machine learning models. Use a standard Git workflow, but
-  without storing large files in the repo. Data is cached by DVC, allowing for
-  efficient sharing. Think "Git for data".
+- **Data Management** - Track and version large amounts of data along with your
+  code, and use DVC as a build-system for reproducible, data driven pipelines.
 
-- **[Data and model access]** goes over using data artifacts from outside of the
-  project and importing them from another DVC project. This can help to download
-  a specific version of an ML model to a deployment server or import a dataset
-  into another project.
+- **Experiment Management** - Live-track your experiments via code
+  instrumentation for light-weight experiment tracking. Learn how to Leverage
+  pipelines as a powerful Git based build system with native parameters, metrics
+  and plots support.
 
-- **[Data pipelines]** describe how models and other data artifacts are built,
-  and provide an efficient way to reproduce them. Think "Makefiles for data and
-  ML projects" done right.
+The following chapters are grouped into the above 2 trails and are all pretty
+self-contained.
 
-- **[Metrics, parameters, and plots]** can be attached to pipelines. These let
-  you capture, evaluate, and visualize ML projects without leaving Git.
+<admon type="tip">
+
+Feel free to "choose your own adventure" and skip to the chapters which answer
+your specific needs. In case you're unsure where to start, we recommend going
+over the chapters in order.
+
+</admon>
+
+## Data Management
```

Comment:
WDYT about not mentioning the subpages at this point and just directing the users to the first page of each "trail"?

I think it would :
- reduce verbosity
- save users from having to choose where to start
- save potential confusion/discussion about the subtitles

Reply:
That's an interesting idea, but I think It would behave more consistently only if we actually unify data/exp to 1 page each (yes, it would be long and require scrolling.... but that's fine, who said a lot of short pages is better)
Ok if I try this in an individual followup PR and we'll give it a try and get a feel for it?

Reply:
No matter if we consolidate or not (again, we had it before in different ways and forms already :) ) - I agree with @daavoo - tbh not clear why we should mention individual subpages. Makes it too lengthy.

Reply:
@shcheklein  - I think it is coupled with single/non-single page honestly, because:
- now chapters are self contained to a sense, you an drop in any of them. Making only 1 chapter an entry point makes the next ones automatically non-entry points and that would require most content changes, it will basically be 1 piece of content per trail, either single page or some wizard with steps (pages), but it would make it so no one would drop on the non-first chapter, so no point in keeping them all self-contained.
 most people don't notice the next/prev buttons, and navigate by links, so they might not notice and miss those nest pages without proper hooks (press next to read about...)

Reply:
Again, if no strong objections, let's leave this to a followup PR please

Reply:
no strong opinion, I see your point, I would still decouple it (from a single page decision, there was no intention to do it this way initially, I'm not sure it's 100% done this way, etc) and simplify the page. If get started starts with a 2-3 screen page just to navigate it - something is wrong with it fundamentally. 

To clarify, I think there are many ways to keep even all these links, give ppl about the structure _and_ keep it preferably ~1 screen. (e.g. reduce explanations, embed aggressively into paragraphs on the main trails explanation, put a list of links one by one after the trail desc, to cards, etc, etc).

Anyways, to clarify, not a blocker on my end.